net/ipv4/tcp_ipv4.c

```diff
@@ -2421,6 +2421,22 @@ static void tcp_v4_clear_sk(struct sock *sk, int size)
 
 	memset(&tp->tk_table.pprev, 0, size - offsetof(struct tcp_sock, tk_table.pprev));
 }
+
+void tcp_copy_sk(struct sock *nsk, const struct sock *osk)
```

Comment:
for maintaining purpose, could you add a comment here? (e.g. _Similar to: sock_copy_)

(and maybe an additional comment why we need an exception for (MP)TCP)